index.html

```diff
@@ -89,6 +89,212 @@ <h1 class="presentation">
+              <span></span>
+              <p>2015</p>
+            </div>
+            <div class="long_desc">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="work__lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="btn btn-enabled">See project</button>
+          </div>
+        </article>
+        <article class="works">
+          <div class="works__image">
+            <img src="images/mobile/3.png" alt="work 3" class="img3" />
+          </div>
+          <div class="work__description">
+            <h2>Tonic</h2>
+            <div class="short_desc">
+              <h4>CANOPY</h4>
+              <span></span>
+              <h4>Back end Dev</h4>
+              <span></span>
+              <p>2015</p>
+            </div>
+            <div class="long_desc">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="work__lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="btn btn-enabled">See project</button>
+          </div>
+        </article>
+        <article class="works">
+          <div class="works__image">
+            <img src="images/mobile/4.png" alt="work 4" class="img4" />
+          </div>
+          <div class="work__description">
+            <h2>Multi-Post Stories</h2>
+            <div class="short_desc">
+              <h4>CANOPY</h4>
+              <span></span>
+              <h4>Back end Dev</h4>
+              <span></span>
+              <p>2015</p>
+            </div>
+            <div class="long_desc">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="work__lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="btn btn-enabled">See project</button>
+          </div>
+        </article>
+      </section>
+      <section class="about-me" id="about">
+        <div class="about-me-container">
+          <div class="about-me__title">
+            <h2>About Myself</h2>
+          </div>
+          <div class="about-me__description">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
```

Comment:
- [ ] Please ensure that the padding at the top of this section has enough room. At the moment, it does not exactly match the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/158474990-22125383-e70a-4f2d-8165-3e9d4fde1ca0.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/158474929-0c21f23b-5f6e-4021-99ca-01b88f3097d1.png)